dags/vz_afd_ems_import.py

```diff
@@ -0,0 +1,84 @@
+# stuff to make the airflow, 1Password integration work
+import os
+import pendulum
+import docker
+from airflow.decorators import dag, task
+from onepasswordconnectsdk.client import Client, new_client
+
+DEPLOYMENT_ENVIRONMENT = os.environ.get("ENVIRONMENT", 'development')   # our current environment from ['production', 'development']
+
+ENVIRONMENT = {
+    "ENVIRONMENT": DEPLOYMENT_ENVIRONMENT,
+    "OP_API_TOKEN": os.getenv("OP_API_TOKEN"),
+    "OP_CONNECT": os.getenv("OP_CONNECT"),
+    "OP_VAULT_ID": os.getenv("OP_VAULT_ID"),
+}
+
+# EMS DAG
+
+# define the parameters of the DAG
+@dag(
+    dag_id="vz-ems-import",
+    description="A DAG which imports EMS data into the Vision Zero database.",
+    schedule="0 7 * * *",
+    start_date=pendulum.datetime(2023, 1, 1, tz="UTC"),
+    catchup=False,
+    tags=["repo:atd-vz-data", "vision-zero", "ems", "import"],
+)
+
+def etl_ems_import():
+
+    @task()
+    def run_ems_import_in_docker():
```

Comment:
based on my current understanding of DAG parsing, i believe you would ideally move `import docker` inside this function.

Reply:
This feels like a TIL coming for me today. I am going to learn about the differences of importing outside of the global scope as it pertains to DAG parsing, and surely move that import down to inside the main DAG function.

Reply:
That is super helpful, John, thank you! I've pulled that import down into the function like you suggested. TIL for sure!!